### PR TITLE
[prman.dd] [intro.dd] [spec.ddoc] Few small fix.

### PR DESCRIPTION
**Pull request by devmynote**
[prman.dd] Add SUBNAV.
[intro.dd] Add $(HEADERNAV_TOC). fix to LNAME2.
[spec.ddoc] Fix SUBNAV_HEAD. (Required for 'narrow layouts') And avoid numbering in '.subnav'.

### CODE REVIEW

**Comment by dkorpel** [comment]
Can you please describe the issues that these changes fix?

**Comment by devmynote** [comment]
Three trivial problems with navigator.

1.[prman.dd]
This page has no navigation compared to other 'foundation' pages.
To fix it, Added 'SUBNAV=$(SUBNAV_FOUNDATION)'.

2.[intro.dd]
This page has no 'Contents' list compared to other 'spec' pages.
To fix it, Added '$(HEADERNAV_TOC)'.
Also, since the section link is incorrect, fixed to 'LNAME2' like any other 'spec' page.

3.[spec.ddoc]
Problems with the 'Narrow layout' defined in 'style.css'.
Navigation is not displayed on all 'spec' pages in 'Narrow layout'.
Therefore,
    'SUBNAV_HEAD' fixed from 'H5' to 'H2' by referring to other pages.
    Fixed to '.hyphenate h2 :: before' to avoid numbering in navigation.

File: spec/intro.dd [hunk 1]
```diff
@@ -10,7 +12,7 @@ to create native libraries or executables.)
 $(P This document is the reference manual for the D Programming Language. For more information and
 other documents$(COMMA) see $(LINK2 https://dlang.org/, The D Language Website).)
 
-$(H2 Phases of Compilation)
+$(H2 $(LNAME2 Phases-of-Compilation, Phases of Compilation))
```
Comment (ntrel) [inline spec/intro.dd:15]: Normally we use lowercase for anchors.

Comment (devmynote) [reply spec/intro.dd]: Thank you. Fixed the anchor to lowercase.

**Review by dkorpel — APPROVED** [review]

**Pull request merged**